.clog.toml

```diff
@@ -0,0 +1,6 @@
+[clog]
+repository = "https://github.com/ramn/metrics_cloudwatch"
+
+changelog = "CHANGELOG.md"
+
+from-latest-tag = true
```

Comment:
Does this require us to tag releases?

Reply:
Yes, but cargo release does that automatically.